algosdk/future/transaction.py

```diff
@@ -1650,23 +1651,24 @@ def teal_bytes(teal):
         ), "Program {} is not bytes".format(teal)
         return teal
 
+    @staticmethod
```

Comment:
I'm not sure if this should be a static method on the `ApplicationCallTxn` class, but I kept it anyway (path of least resistance).

Reply:
Maybe this can just be a standalone function in `algosdk/encoding.py` ?

Reply:
👍